src/features/list-view/ListView.js

```diff
@@ -14,7 +14,9 @@ type Props = {
         rowIndex: number,
     |}) => string | React.Node,
     getGridHeader: (columnIndex: number) => any,
+    getGridHeaderSort?: (columnIndex: number) => React.Node | null,
```

Comment:
Thoughts around moving the chevron into ListView and switch the return type to ASC/DESC/null?